tools/markdown-checker/lib/transformer/traverser.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {Object} Node 
+ * @param {function} callBackLogic 
+ */
+
+ module.exports = function traverseChildren(node, callBackLogic) {
+  node.forEach(n => {
+    callBackLogic();
```

Comment:
`callBackLogic` is `undefined` in recursive calls

Reply:
Development is still going on...